rllib/models/torch/fcnet.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, obs_space: gym.spaces.Space,
                               model_config, name)
         nn.Module.__init__(self)
 
-        hiddens = model_config.get("fcnet_hiddens", []) + \
+        hiddens = list(model_config.get("fcnet_hiddens", [])) + \
             model_config.get("post_fcnet_hiddens", [])
```

Comment:
```suggestion
            list(model_config.get("post_fcnet_hiddens", []))
```
, since `post_fcnet_hiddens` will be tuple if we use HyperOpt to search its best value (similar to `fcnet_hiddens`).  (detail in #14304)